.circleci/config.yml

```diff
@@ -82,7 +68,7 @@ jobs:
       - run: npm install
       - run: npm run build
       - run: sudo npm install -g broken-link-checker-local
-      - run: blcl -r --exclude "/content-guide/" --exclude "/webmasters/tools/" --exclude-external ./site
+      - run: sudo blcl -r --exclude "/content-guide/" --exclude "/webmasters/tools/" --exclude-external ./site
```

Comment:
@sirleech so this fixes those other broken attempts from the last group of commits? I haven't used the link checker before.